src/luxonis_ml/data/dataset.py

```diff
@@ -318,7 +324,7 @@ def _start_time(self) -> None:
 
     def _end_time(self) -> None:
         self.t1 = time.time()
-        print(f"Took {self.t1 - self.t0} seconds")
+        logging.info(f"Took {self.t1 - self.t0} seconds")
```

Comment:
Better to use `self.logger.info(...)` instead of `logging.info(...)` (the former contains additional info).